drone.sh

```diff
@@ -119,13 +124,21 @@ do
     PORT=$DEFAULT_PORT
     fi
     echo $(printf "%s" "$ $(printf "$expr" "$PORT") $USER@$HOST:$PLUGIN_TARGET ...")
+    if [ -n "$PLUGIN_PRESCRIPT" ]; then
+        echo $(printf "%s" "$ ssh -p $PORT $USER@$HOST ...")
+        echo $(printf "%s" " > $prescript ...")
+        eval "ssh -p $PORT $USER@$HOST '$prescript'"
+        result=$(($result+$?))
+        echo $(printf "%s" "$ ssh -p $PORT $USER@$HOST result: $?")
+        if [ "$result" -gt "0" ]; then exit $result; fi
+    fi
     eval "$(printf "$expr" "$PORT") $USER@$HOST:$PLUGIN_TARGET"
     result=$(($result+$?))
     if [ "$result" -gt "0" ]; then exit $result; fi
-    if [ -n "$PLUGIN_SCRIPT" ]; then
+    if [ -n "$PLUGIN_POSTSCRIPT" ]; then
```

Comment:
The removal of the `script` option will break every existing usage of this plugin that used that option. I'd love to see a way to keep that functionality intact.

Reply:
returned to PLUGIN_SCRIPT to avoid breaking